collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormFinalizingTest.kt

```diff
@@ -21,27 +22,26 @@ class FormFinalizingTest {
     val copyFormChain: RuleChain = chain().around(rule)
 
     @Test
-    fun fillingForm_andPressingSaveAsDraft_doesNotFinalizesForm() {
+    fun fillingForm_andPressingFinalize_finalizesForm() {
         rule.startAtMainMenu()
             .copyForm(FORM)
             .assertNumberOfFinalizedForms(0)
             .startBlankForm("One Question")
-            .swipeToEndScreen()
-            .clickSaveAsDraft()
-            .assertNumberOfEditableForms(1)
-            .assertNumberOfFinalizedForms(0)
+            .fillOutAndFinalize(FormEntryPage.QuestionAndAnswer("what is your age", "52"))
+            .assertNumberOfEditableForms(0)
+            .clickSendFinalizedForm(1)
```

Comment:
There is no need to click on the button just use `assertNumberOfFinalizedForms` like in the test below.